src/app/search/search-filters.ts

```diff
@@ -1253,6 +1253,9 @@ function searchFilters(
               (socket) =>
                 socket &&
                 socket.plug &&
+                socket.plug.plugItem &&
+                socket.plug.plugItem.plug &&
+                socket.plug.plugItem.plug.plugCategoryHash &&
```

Comment:
This can now be written as `idx(socket, (s) => s.plug.plugItem.plug.plugCategoryHash)`